add_gif.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Usage: ./add_gif.sh name http://url_of_the_gif
+NAME=$1
+URL=$2
+
+# Usage 
+if [ "$NAME" = "--help" ]; then
```

Comment:
(but 2, since `$#` is a count of the arguments, i.e. excludes the command itself)